go.mod

```diff
@@ -15,11 +16,12 @@ require (
 	github.com/joshcarp/mermaid-go v0.0.0-20200403052241-d6fecd5216de
 	github.com/pkg/errors v0.9.1
 	github.com/radovskyb/watcher v1.0.7
-	github.com/russross/blackfriday v2.0.0+incompatible
+	github.com/russross/blackfriday/v2 v2.0.1
```

Comment:
v2 doesn't work with go mods: https://github.com/russross/blackfriday/issues/500
Already talked with @cuminandpaprika about this

Reply:
👍 Removed